include/convex_bodies/hpolytope.h

```diff
@@ -909,6 +908,18 @@ class HPolytope {
             v += a;
     }
 
+    template <typename update_parameters>
+    NT compute_reflection(Point &v, const Point &, MT const &AE, VT const &AEA, NT const &vEv, update_parameters const &params) const {
+
+            Point a((-2.0 * params.inner_vi_ak) * A.row(params.facet_prev));
+            VT x = v.getCoefficients();
+            NT new_vEv = vEv - (4.0 * params.inner_vi_ak) * (AE.row(params.facet_prev).dot(x) - params.inner_vi_ak * AEA(params.facet_prev));
```

Comment:
since v' = v - 2 <v,a_i> a_i, the new energy is (v - 2 <v,a_i> a_i)^T E (v - 2 <v,a_i> a_i).
I think the new energy should be: 
`NT new_vEv = vEv - (4.0 * params.inner_vi_ak) * (AE.row(params.facet_prev).dot(x)  + 4.0 * params.inner_vi_ak * AEA(params.facet_prev));`